zlp-scheduler/app/views/admin/add_student.html.haml

```diff
@@ -0,0 +1,32 @@
+%div#addAdminDiv  
+    %h1{:style => "text-align:center"} Add a new Student to #{@cohort_name}
+    = form_for @user,url: add_student_path, method: "post" do |f| 
+        %table#userTable
+            %tr
+                %th#specialTableHeader First Name
+                %th#specialTableHeader Last Name
+                %th#specialTableHeader TAMU Email
```

Comment:
You can set both email and UIN to `required => true`

Reply:
Fixed